.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": ["biome:recommended"]
+}
```

Comment:
Maybe I'm missing something but it seems that you are not defining the necessary dependencies for this config.

And if you are going to use biome rules... why don't use biome and drop eslint and prettier. :D